lib/index.js

```diff
@@ -47,7 +50,7 @@ function toTypeString (value) {
   return Object.prototype.toString.call(value).slice(8, -1)
 }
 
-module.exports = (secp256k1) => {
+module.exports = (secp256k1, bitcoinerlab) => {
```

Comment:
why doesn't the bitcoinerlab one have a compatible API? 🤔

Reply:
`bitcoinerlab` has `tiny-secp256k1` api